packages/dso-toolkit/src/components/accordion/accordion.scss

```diff
@@ -240,6 +240,43 @@
   }
 }
 
+// Create space between sections of the default accordion
+// ------------------------------------------------------
+:where(.dso-accordion-section) {
```

Comment:
Wat is het nut van een `:where()` met slechts 1 selector? Op regel 259 selecteer je hetzelfde element maar dan zonder `:where()` selector, waarom?

En gevolgd door `& + &`  vraag ik mij af of het er nou heel veel leesbaarder van wordt.

Heeft het iets met specificity te maken? Op regel 252 gebruik je ook `:where()` met 1 selector.

Reply:
@hansgrimm heb je deze vraag voorbij zien komen?

Reply:
Nee 😉

Reply:
`:where` zorgt voor een specificity van 0. Altijd goed, ook met maar 1 selector.
Waar ik ‘m niet gebruik, is dat om de selector sterker te maken.

Reply:
OK, dank!